util/limbo_compat.h

```diff
@@ -114,7 +114,16 @@ using namespace godot;
 #define ADD_STYLEBOX_OVERRIDE(m_control, m_name, m_stylebox) (m_control->add_theme_stylebox_override(m_name, m_stylebox))
 #define GET_NODE(m_parent, m_path) m_parent->get_node_internal(m_path)
 #define OBJECT_DB_GET_INSTANCE(m_id) ObjectDB::get_instance(m_id)
-#define EDITOR_DEF(m_setting, m_value) EditorInterface::get_singleton()->get_editor_settings()->set_initial_value(m_setting, m_value, false)
+#ifdef LIMBOAI_MODULE
+	#define EDITOR_DEF(m_setting, m_value) EditorInterface::get_singleton()->get_editor_settings()->set_initial_value(m_setting, m_value, false)
+#elif LIMBOAI_GDEXTENSION /* for gdextension, we apparently have to set the setting once to be available. TODO: remove case on feature parity. */
```

Comment:
We're already inside the `#ifdef LIMBOAI_GDEXTENSION` (line 74).

Reply:
The limbo_compat.h file is split into two parts: the top part is for module, and the bottom part is for the extension. This code is inside the part that is meant for the extension. There is also a third part that is for code that is compatible with both (API abstractions).

Reply:
I was blindly tracing code. Sorry. I pushed the removal of duplicated LIMBOAI_GDEXTENSION/-MODULE guards.
If you prefer some other solution, let me know.

Reply:
Looks good to me